includes/base/controls-stack.php

```diff
@@ -2070,6 +2074,9 @@ public function print_template() {
 		<script type="text/html" id="tmpl-elementor-<?php echo esc_attr( $this->get_name() ); ?>-content">
 			<?php $this->print_template_content( $template_content ); ?>
 		</script>
+		<script type="text/html" id="tmpl-elementor-<?php echo esc_attr( $this->get_name() ); ?>-content-single">
+			<?php $this->single_content_template(); ?>
+		</script>
```

Comment:
Try to keep this logic in `Widget_Nested_Base`. Meaning override `print_template()` in `Widget_Nested_Base`.

Also you are now ALWAYS adding `<script>` tags, even if it will be empty. Only add these script tags if the widgets supports it.